libkbfs/journal_server.go

```diff
@@ -93,6 +98,22 @@ func makeJournalServer(
 	return &jServer
 }
 
+func (j *JournalServer) getDirLocked() string {
+	// TODO: This may end up making paths too long for some
+	// systems. We may end up having to drop the UID (since a
```

Comment:
Tests still don't pass, but went with truncating the KID string, which is easier than plumbing through a separate device ID and has the same properties. What do you think?